vividus-plugin-azure-storage-account/src/main/java/org/vividus/azure/storage/blob/BlobStorageSteps.java

```diff
@@ -114,9 +116,17 @@ public void downloadBlob(String blobName, String containerName, String storageAc
     {
         try (ByteArrayOutputStream outputStream = new ByteArrayOutputStream())
         {
-            blobServiceClientFactory.createBlobClient(blobName, containerName, storageAccountKey)
-                    .downloadStream(outputStream);
-            variableContext.putVariable(scopes, variableName, outputStream.toString(StandardCharsets.UTF_8));
+            var blobClient = blobServiceClientFactory.createBlobClient(blobName, containerName, storageAccountKey);
```

Comment:
we don't use `var` in production code yet, as we don't have clear strategy for it, it's allowed in test code only

Reply:
fixed